src/include/sof/trace/trace.h

```diff
@@ -435,42 +439,59 @@ struct tr_ctx {
 
 /* tracing from infrastructure part */
 
-#define tr_err(ctx, fmt, ...) \
-	trace_error_with_ids(_TRACE_INV_CLASS, ctx, \
-			     _TRACE_INV_ID, _TRACE_INV_ID, fmt, ##__VA_ARGS__)
-
 #define tr_err_atomic(ctx, fmt, ...) \
 	trace_error_atomic_with_ids(_TRACE_INV_CLASS, ctx, \
 				    _TRACE_INV_ID, _TRACE_INV_ID, \
 				    fmt, ##__VA_ARGS__)
 
-#define tr_warn(ctx, fmt, ...) \
-	trace_warn_with_ids(_TRACE_INV_CLASS, ctx, \
-			    _TRACE_INV_ID, _TRACE_INV_ID, fmt, ##__VA_ARGS__)
-
 #define tr_warn_atomic(ctx, fmt, ...) \
 	trace_warn_atomic_with_ids(_TRACE_INV_CLASS, ctx, \
 				   _TRACE_INV_ID, _TRACE_INV_ID, \
 				   fmt, ##__VA_ARGS__)
 
-#define tr_info(ctx, fmt, ...) \
-	trace_event_with_ids(_TRACE_INV_CLASS, ctx, \
-			     _TRACE_INV_ID, _TRACE_INV_ID, fmt, ##__VA_ARGS__)
-
```

Comment:
Why are all the non-atomic variants being removed?

Reply:
They are moved, not removed. check a little below, they are moved to the #else part.